module/zfs/arc.c

```diff
@@ -5702,6 +5702,7 @@ arc_read(zio_t *pio, spa_t *spa, const blkptr_t *bp,
 		 * Gracefully handle a damaged logical block size as a
 		 * checksum error.
 		 */
+		ASSERT3U(lsize, <=, spa_maxblocksize);
```

Comment:
Missing function parameter list.
Should be:
`ASSERT3U(lsize, <=, spa_maxblocksize(spa));`